OPHD/States/MapViewStateEvent.cpp

```diff
@@ -89,6 +89,7 @@ void MapViewState::onFactoryProductionComplete(Factory& factory)
  */
 void MapViewState::onDeployColonistLander()
 {
+	mTurnNumberOfLanding = mTurnCount;
```

Comment:
That's nice and simple.

I suppose I should bring up a section from the last review:
> We would still need to retain the second check, since there can be multiple landers, and we probably want to activate on the first landing, and not reset the value upon a subsequent landing.

I'm fine with resetting `mTurnNumberOfLanding` on a subsequent landing. There wasn't really a clear specification on desired behavior here. Just wanted to draw attention to this in case it was an accidental change.

Or maybe @ldicker83 would like to weigh in on desired behavior in this case. Should the field represent the turn of the first landing, or the turn of the last landing? With multiple landers, the two can be different.

Reply:
Whoops, I did miss this. Thank you for pointing that out. Since you've asked for confirmation I'll wait to amend it until that's answered, but it is unintentional on my part.

Reply:
Doesn't seem like Leeor is able to respond right now. Given that change was accidental, I think maybe just go ahead and re-add the check `if (mTurnNumberOfLanding > mTurnCount)` around that line. I think that's probably what everyone is expecting.

Plus, it avoids a possible case where morale can become active due to an early landing, and then gets disabled again because more colonists land, leaving morale paused in a slightly altered state. That might be a bit unexpected.

----

Alternatively, I suppose you could use [`std::min`](https://en.cppreference.com/w/cpp/algorithm/min) instead of an `if` check:
```cpp
mTurnNumberOfLanding = std::min(mTurnNumberOfLanding, mTurnCount);
```

Reply:
Done.